src/navigation/screenOptions.tsx

```diff
@@ -67,9 +67,10 @@ const styles = StyleSheet.create({
     lineHeight: normalize(29)
   },
   headerTitleContainerStyle: {
-    flex: 1,
+    flex: 500,
```

Comment:
Why is the "flex: 500" style important to us? Why did you make this change?

Reply:
And is there any particular reason why it's 500? Maybe 400? Or 0? Is it written in a document that we need to make the flex value 500?

Reply:
there was a slight difference between 400 and 500 and I figured out that it has no negative effects if the flex value is as big as this so I chose 500.
Flex 0 is also a option but it centers the header and is not positioned more on the left side like before:
| Flex 0 | Flex 500 |
| ------------- | ------------- |
| ![simulator_screenshot_A6CF4E25-7AA9-4429-A4F1-51BE8C62CFA2](https://github.com/smart-village-solutions/smart-village-app-app/assets/105894697/7ec90510-68db-4a5f-8e47-744645467ae8) |  |

Reply:
what about no flex at all and removing `, alignItems: 'center'`?

Reply:
seems to be the same result as with `flex: 0`

Reply:
but the main issue is the spacing in the header with some buttons in some screens:
| Flex 0 / no Flex | Flex 500 |
| ------------- | ------------- |
| ![Simulator Screenshot - iPhone 14 Pro - 2023-08-23 at 10 56 56](https://github.com/smart-village-solutions/smart-village-app-app/assets/105894697/c0973e57-fa72-4082-9ff1-39c1bc4c5f0e) | ![Simulator Screenshot - iPhone 14 Pro - 2023-08-23 at 10 57 19](https://github.com/smart-village-solutions/smart-village-app-app/assets/105894697/c28725d7-1cd6-4d90-a5b2-48bc48e3efe8) |